sdk/servicebus/azure-servicebus/azure/servicebus/_servicebus_client.py

```diff
@@ -214,6 +210,56 @@ def get_queue_receiver(self, queue_name, **kwargs):
             **kwargs
         )
 
+    def get_queue_deadletter_receiver(self, queue_name, **kwargs):
+        # type: (str, Any) -> ServiceBusReceiver
+        """Get ServiceBusReceiver for the sub Dead Letter Queue of the specific Queue.
```

Comment:
Wording clarity concern:  "sub dead letter queue" isn't super clear to my ears.  The queue under the dead letter queue?

I'm thinking a simple tweak to "Dead letter sub-queue" might help but that only slightly improves the aformentioned confusion and isn't perfect.  Feel free to chime in if you see any better options; I've not seen any other obvious answers.

Reply:
copy some words from introduction in the service doc: https://docs.microsoft.com/en-us/azure/service-bus-messaging/service-bus-dead-letter-queues

feel free to give git suggestions and I'll apply the suggestion :)

Reply:
I like your rewording.  I was contemplating whether "provided implicitly" would add more clarity to that you get the DLQ for free under a given queue; as well as whether "cannot be processed" is clear in terms of failures while processing, but we've hit the diminishing returns on my nitpicking and I won't waste our time on these unless we see users getting confused, since it's the same verbiage in the docs you pointed out.